examples.md

```diff
@@ -409,22 +409,15 @@ Replace `~/.local/share/renv` with the correct `path` if not using Ubuntu.
 
 ## Ruby - Bundler
 
+[`actions/setup-ruby`](https://github.com/ruby/setup-ruby) already provides a way to automatically run `bundle install` and cache the result:
```

Comment:
This is great but I think we should keep the manual install example too. Reasons being we want this page to be the main place for people to find examples of using cache action and we don't want broken link in case the `setup-ruby` readme changes.

Reply:
I’ve now updated the example to include both the automatic way (using the `bundler-cache` option) as well as the manual way (using `actions/cache`).